packages/excalidraw/laser-trails.ts

```diff
@@ -29,27 +30,32 @@ export class LaserTrails implements Trail {
   }
 
   private getTrailOptions() {
+    const size = 1.5;
+    const DECAY_DELAY = 0;
+    const DECAY_TIME = 3000;
+
     return {
       simplify: 0,
       streamline: 0.4,
-      sizeMapping: (c) => {
-        const DECAY_TIME = 1000;
-        const DECAY_LENGTH = 50;
-        const t = Math.max(
+      size,
+      sizeMapping: () => {
```

Comment:
original sizeMapping would limit the laser trail to 50 points and decay after a second. 
The suggested implementation doesn't limit the size, and only depends on the time elapsed after the mouse was released while drawing the laser figurine.